### PR TITLE
[bitnami/metrics-server] Enable support update strategy for deployment

### PR DESCRIPTION
**Pull request by tylern91**
**Description of the change**

Add a `strategy` field for the metrics-server deployment by defining them in values.yaml

**Benefits**

This allows us rolling update the metrics-server deployment when using multiple replicas.

**Possible drawbacks**

None known

**Applicable issues**

None known

**Additional information**

None

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/metrics-server/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: metrics-server
 sources:
   - https://github.com/bitnami/bitnami-docker-metrics-server
   - https://github.com/kubernetes-incubator/metrics-server
-version: 5.8.15
+version: 5.8.16
```
Comment (miguelaeh) [inline bitnami/metrics-server/Chart.yaml:26]: Please bump the minor version

File: bitnami/metrics-server/templates/deployment.yaml [hunk 2]
```diff
@@ -7,6 +7,9 @@ spec:
   replicas: {{ .Values.replicas }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
```
Comment (miguelaeh) [inline bitnami/metrics-server/templates/deployment.yaml:11]: Please use the common.tplvalue function

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!
Please take a look at my comments

File: bitnami/metrics-server/templates/deployment.yaml (hunk 2 above)
Comment (tai-nguyen-mox) [reply bitnami/metrics-server/templates/deployment.yaml]: Updated by https://github.com/bitnami/charts/pull/6917/commits/464e25525586741f3d3e6fb2cc9401a2fa7a27c9

File: bitnami/metrics-server/Chart.yaml (hunk 1 above)
Comment (tai-nguyen-mox) [reply bitnami/metrics-server/Chart.yaml]: Updated by https://github.com/bitnami/charts/pull/6917/commits/464e25525586741f3d3e6fb2cc9401a2fa7a27c9

**Review by miguelaeh — APPROVED** [review]
Thank you very much for the change!
LGTM!

**Pull request merged**